Source/FakeItEasy.Net35/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,3 @@
+﻿﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("FakeItEasy.Net35.Tests, PublicKey=00240000048000009400000006020000002400005253413100040000010001002f50d82092713be482c885861c984334606da0f54c78738a3dd0862fb2dfc6080f0780132cc65d88f0f0c70af74e8a53430962395bfc1a36fab08b7a2549d387e805c13cc84acd884447ec8c4dcfb6216df720f0998380f9c906b5de8141798d64661f036d47274e6ecb76c9cde5f4cf2b521040601e44b3914fbeb9f39127f9")]
```

Comment:
Yeah, you need the plug-in. It does indeed sounds like a good candidate for a new issue.